pkg/plugin/work-in-progress/event_handler_gh_test.go

```diff
@@ -72,6 +77,16 @@ var _ = Describe("Test Keeper Plugin features", func() {
 			// given
 			NonExistingRawGitHubFiles("work-in-progress.yml", "work-in-progress.yaml")
 
+			gock.New("https://api.github.com").
+				Get("/repos/bartoszmajsak/wfswarm-booster-pipeline-test/issues/4/labels").
+				Reply(200).
+				BodyString("[]")
+
+			gock.New("https://api.github.com").
+				Post("/repos/bartoszmajsak/wfswarm-booster-pipeline-test/issues/4/labels").
+				Reply(200).
```

Comment:
all the `Post` mocks should have a matcher?
eg:
```gogock.New("https://api.github.com").
	Post("/repos/bartoszmajsak/wfswarm-booster-pipeline-test/issues/4/labels").
	SetMatcher(ExpectContentEqualTo(`work-in-progress`)).
	Reply(200).
	Body(FromFile("test_fixtures/github_calls/wip_pr_created_with_label.json"))
```
in the `ExpectContentEqualTo` you would need to retrieve the content and match it

Reply:
Technically, you could also use the current method:
`SetMatcher(ExpectPayload(To(HaveBody(`work-in-progress`)))).`
with the fact that the `ExpectPayload` should be able to unmarshall not only a map, but also an array